assets/css/aboutPage.css

```diff
@@ -73,6 +73,9 @@ summary strong {
     margin-bottom: 40px;
 }
 
+.about-us_faq_answers {
+    padding-left: 20px;
+}
```

Comment:
Hey @john-fowle, could we add:

```
    display: flex;
    flex-direction: column;
```
    
    ... to this class :D Because we wrapped it all up in a new `<div>` our answer started to look a bit different :D 
    
    
![image](https://user-images.githubusercontent.com/51357920/145460770-fa011e52-2611-467d-b176-4e1292d1a970.png)

Reply:
Okay, added that and pushed it. I was afraid that might happen!